examples/resources/cluster/resource_attach_cluster.tf

```diff
@@ -13,5 +13,6 @@ resource "tanzu-mission-control_cluster" "attach_cluster_without_apply" {
     cluster_group = "default" # Default: default
   }
 
+  ready_wait_timeout = "0s"
```

Comment:
can you explain why is this needed ?

Reply:
@shreyassreenivas I asked for this change, basically if we are not attaching the cluster and only creating it (without the attach block) then I don't see any reason to block the terraform apply for 3 minutes.

WDYT

Reply:
Added to the documentation, so that users can take reference and not wait for 3m when kubeconfig is not present.